draft-ietf-moq-transport.md

```diff
@@ -1683,10 +1688,12 @@ TODO: figure out how a relay closes these streams
 When a stream begins with `STREAM_HEADER_TRACK`, all objects on the stream
 belong to the track requested in the Subscribe message identified by `Subscribe
 ID`.  All objects on the stream have the `Publisher Priority` specified in the
-stream header.
+stream header. The Header Length specifies the total encoded length of the header in bytes which includes Subscribe ID, Track Alias and Publisher Priority fields.
+
 
 ~~~
 STREAM_HEADER_TRACK Message {
+  Header Length (i),
```

Comment:
I would call this Stream Header Length to avoid confusion with Object Header Length.